### PR TITLE
Refactor: Add require relative for spec harness

### PR DESCRIPTION
**Pull request by bwbolt**
### What does this PR do?
- This PR add spec-harness access-ability .

### Does this PR break anything?
- [ ] Yes
- [X] No
- If yes explain:

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 require 'csv'
-require './modules/game_statistics'
-require './modules/league_statistics'
-require './modules/season_statistics'
-require './modules/team_statistics'
+require_relative '../modules/game_statistics'
+require_relative '../modules/league_statistics'
+require_relative '../modules/season_statistics'
+require_relative '../modules/team_statistics'
```
Comment (z-prince) [inline lib/stat_tracker.rb:5]: This will make testing a lot easier.

**Review by z-prince — APPROVED** [review]
Well done.

**Review by Hutchins13531 — COMMENTED** [review]
Nice, I'll go ahead an merge!

**Comment by Hutchins13531** [comment]
Looks good, but it'll look better on the main branch... merging now!

**Pull request merged**